templates/workflow/workflow.tpl

```diff
@@ -98,6 +103,16 @@
 				{help file="editorial-workflow/publication" class="pkp_help_tab"}
 				<div class="pkpPublication" ref="publication" aria-live="polite">
 					<pkp-header class="pkpPublication__header" :is-one-line="false">
+						<span class="pkpPublication__changeSubmissionLanguage">
+							<span>{translate key="submission.list.changeSubmissionLanguage.currentLanguage" language="{{ currentSubmissionLanguageLabel }}"}</span>
+							<pkp-button
```

Comment:
I wonder if the disabled "change" button should be completely hidden rather than disabled when a submission has been published. It's not necessarily clear that it's a button at this time either since it's a disabled text button without additional decoration.

Reply:
I asked Devika, lets see what she says...